dbt/models/ccao/ccao.vw_time_util.sql

```diff
@@ -0,0 +1,4 @@
+-- View that supplies dynamic datetime values in a way that we can mock for
+-- unit tests. See:
+-- https://discourse.getdbt.com/t/dynamic-dates-in-unit-tests/16883/2
+SELECT CURRENT_DATE AS date_today
```

Comment:
Kind of a bummer that dbt unit tests don't offer a simpler way of mocking dynamic values, but at least it works.